docs/cristin.ttl

```diff
@@ -151,11 +151,24 @@
     Owing to the number of funding bodies, there is no uniform way to dereference grants externally.
     """ .
 
-:TYPE_BOK_RAPPORT_Class
-    a               :CristinClass ;
-    rdfs:comment    "Indicates that the respective PRESENTASJON_VARBEID_Class instance is a reference to a Cristin project" ;
-    rdfs:domain     :Result ;
-    rdfs:subClassOf :PRESENTASJONSTYPEKODE_VALUES_Class .
```

Comment:
This definition is in the document twice, and this one here is faulty, so I'm removing it.